src/api/dev.ts

```diff
@@ -120,7 +120,7 @@ class Watcher extends EventEmitter {
 		// remove this in a future version
 		const template = read_template(src);
 		if (template.indexOf('%sapper.base%') === -1) {
-			const error = new Error(`As of Sapper v0.10, your template.html file must include %sapper.base% in the <head>`);
+			const error: NodeJS.ErrnoException = new Error(`As of Sapper v0.10, your template.html file must include %sapper.base% in the <head>`);
```

Comment:
At what point do we just remove this error entirely?

It's been a long, long time since 0.10 and we don't have these sorts of protections around other breaking changes.

Reply:
yeah, good point. I've gone ahead and removed it